src/test/java/unit/BqDatasetControlled.java

```diff
@@ -262,25 +263,51 @@ void createWithAllOptions() throws IOException {
     TestCommand.runCommandExpectSuccess("resources", "delete", "--name=" + name);
   }
 
-  /** Helper method to call `terra resources list` and expect one resource with this name. */
+  /**
+   * Helper method to call `terra resources list` and expect one resource with this name. Uses the
+   * current workspace.
+   */
   static UFBqDataset listOneDatasetResourceWithName(String resourceName)
       throws JsonProcessingException {
-    List<UFBqDataset> matchedResources = listDatasetResourceWithName(resourceName);
+    return listOneDatasetResourceWithName(resourceName, null);
```

Comment:
This method definition is the overload with just one argument so that other places don't have to pass `null`. Or maybe I'm misunderstanding -- can you restate?

Reply:
Oh  I read it wrong. Never mind.